Scripts/SecondsToString/SecondsToString.yml

```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: SecondsToString
+  version: -1
+name: SecondsToString
+script: ''
+type: python
+tags:
+- transformer
+- date
```

Comment:
i think this should be number, no?

Reply:
I don't think so.  It's for converting a date-type field so it belongs in the date section.  The fact that the input is actually a number is incidental as it's just an epoch timestamp.

Reply:
Sorry, my reply was rubbish -- wasn't thinking properly, not an epoch date at all.  I do think it still belongs in the date section, though, as it relates to time.  But at the end of the day, I don't really care.